helper-cli/src/main/kotlin/common/PackageConfigurationOption.kt

```diff
@@ -21,6 +21,8 @@ package org.ossreviewtoolkit.helper.common
 
 import org.ossreviewtoolkit.model.utils.PackageConfigurationProvider
 import org.ossreviewtoolkit.model.utils.SimplePackageConfigurationProvider
+import org.ossreviewtoolkit.utils.ORT_PACKAGE_CONFIGURATIONS_FILENAME
+import org.ossreviewtoolkit.utils.ortConfigDirectory
```

Comment:
The helper CLI is not using ORTs configuration as default.
Why should it use it only for the package configurations?

Reply:
I just wanted to align the code with `cli`, but I'm more than happy to drop this.

Reply:
Would you mind sharing why?

Reply:
Why the alignment or why I'm happy to drop the change?

Reply:
Why you're happy to drop the change.

Reply:
Because I generally do not feel comfortable touching `helper-cli`'s code. It's a tool that arose mostly from HERE's specific needs during support, it seems, and I don't know anyone outside of HERE using it. If we had binary releases for ORT, I'd even argue `helper-cli` should be moved to its own repository. But that's a different story.

Reply:
No-one outside of HERE is using the `helper-cli` since it not publicly documented - something on my todo list. I have showed it to several of our users in support calls.

Reply:
> The helper CLI is not using ORTs configuration as default.
> Why should it use it only for the package configurations?

Apart from the inconsistency it seems to be reasonable that the helper-cli reads the configuration files from the same location as the main CLI by default. @fviernau As the main author of helper-cli, would you agree, or why not? Because if the answer is yes I think it would be better to keep that change and align the default location for the other config files as well.

> If we had binary releases for ORT, I'd even argue helper-cli should be moved to its own repository.

I think the problem with that would be that as long as we don't have and use stable releases moving the code to another repository would make it unmaintainable. Shall we add the helper-cli to the agenda of the next TSC meeting?

Reply:
I believe before that we should actually talk about binary releases in general, incl. things like versioning scheme etc., because depending on that different solutions may be viable for `helper-cli` and other "consumers" of ORT core modules. But yes, please let's put binary releases on the TSC agenda.

Reply:
> Apart from the inconsistency it seems to be reasonable that the helper-cli reads the configuration files from the same location as the main CLI by default.

> As the main author of helper-cli, would you agree, or why not? 

I think doing so is not totally for free, implementation takes time, it adds complexity so usage will be more complex as well. 
I don't currently see that adding much / any value, and I don't see a reason for deciding it right now. So I prefer deferring that decision until it becomes clear that this is the right thing to do.

> I think the problem with that would be that as long as we don't have and use stable releases moving the code to another repository would make it unmaintainable. Shall we add the helper-cli to the agenda of the next TSC meeting?

I don't have any need talking about moving it to another repo, so no need from my side to put it onto the agenda.